src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -155,5 +155,10 @@ def add_records(
         else:
             self._records = records
 
+    def delete_records(self, *args, **kwargs):
+        raise NotImplementedError(
+            "Deleting records for local datasets can be done by deleting record from the list FeedbackDataset._records."
```

Comment:
If we are planning to point users to `_records` then there's no point on making it protected, I think we can just make it public